.github/workflows/warm-up-vercel.yml

```diff
@@ -0,0 +1,32 @@
+## Vercel uses AWS Lambda. When we call getStaticProps in fallback mode or getServerSideProps,
+## a cold Lambda function may take 5-15 seconds to execute. This pipeline warms up Lambda
+## containers for new deployments and keeps them up by fetching affected URLs every 10 minutes.
+
+name: Warm-up Vercel
+
+on:
+  deployment_status:
+  schedule:
+    - cron: "*/10 * * * *"
```

Comment:
This event will work only after we merge the PR. If anything is broken, I’ll submit a quick follow-up PR with a fix.

List of workflow runs: https://github.com/blockprotocol/blockprotocol/actions/workflows/warm-up-vercel.yml